uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -549,6 +549,10 @@
                                     left: 3px;
                                     position: relative;
                                     top: 0;
+                                    /* increase clickable size of dropdown toggle */
+                                    padding: 0 5px 5px;
+                                    margin: 0 -5px -5px;
+
```

Comment:
On smaller screen sizes the flyout menu appears instead, having the increased size caused regressions on the flyout so I limited it to the screensizes before flyout appears.